test/e2e/stats_test.go

```diff
@@ -264,4 +264,38 @@ var _ = Describe("Podman stats", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(limit).To(BeNumerically("==", 100*1024*1024))
 	})
+
+	It("podman stats by default get all running containers", func() {
+		/*
+		   Executing two containers: top-container-1 and ls-container-1.
+		   The latter will exit immediately and the former will continue running.
+		   Expecting to see only one running container in stats and not the one which exited
+		*/
+		session := podmanTest.RunTopContainer("top-container-1")
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		lsSession, _, _ := podmanTest.RunLsContainer("ls-container-1")
```

Comment:
please change this to podman create ...
Running a container is not needed here and more expensive.